docs/streams/stream-examples/realtime-updates.md

```diff
@@ -1,32 +1,201 @@
+import Prerequisites from '../../_partials/_prerequisites-sdk-api-key.md';
+import Steps from '../../_partials/_get-started-steps.md';
+import ConnectToGDN from '../../_partials/_connect-to-gdn-code-block.md';
 
-This tutorial is about using Macrometa GDN as a realtime database with local latencies across the globe.
+This tutorial demonstrates how to use Macrometa GDN as a real-time database with local latencies across the globe.
 
-## Prerequisites
+<Prerequisites />
 
-- A Macrometa account with sufficient permissions to create streams.
-- An API key. For more information, refer to [Create API Keys](../../account-management/api-keys/create-api-keys).
-- Appropriate SDK installed. For more information, refer to [Install SDKs](../../sdks/install-sdks.md).
+## Code
 
-## Realtime Updates Code
+This page guides you through creating a collection, subscribing to the collection, and automatically adding and deleting data to the collection.
+
+<Steps />
+
+If you want to skip the explanation and just run the code, then go directly to the [Full Demo File](#full-demo-file).
+
+### Step 1. Connect to GDN
+
+To update a database with Macrometa Global Data Network (GDN), you must first establish a connection to a local region.
+
+<ConnectToGDN />
+
+### Step 2. Set Variables
+
+Set the variables required to run the code.
 
 <Tabs groupId="operating-systems">
-<TabItem value="js" label="Javascript">
+<TabItem value="js" label="JavaScript">
+
+```js
+// Variables
+const collectionName = "ddos";
+let listener;
+const data = [
+  { ip: "10.1.1.1", action: "block", rule: "blocklistA" },
+  { ip: "20.1.1.2", action: "block", rule: "blocklistA" },
+  { ip: "30.1.1.3", action: "block", rule: "blocklistB" },
+  { ip: "40.1.1.4", action: "block", rule: "blocklistA" },
+  { ip: "50.1.1.5", action: "block", rule: "blocklistB" }
+];
+
+const sleep = (milliseconds) => {
+  return new Promise(resolve => setTimeout(resolve, milliseconds));
+};
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+import threading
+import pprint
+import time
+COLLECTION_NAME = "ddos"
+
+# Variables - Data
+data = [
+    {"ip": "10.1.1.1", "action": "block", "rule": "blocklistA"},
+    {"ip": "20.1.1.2", "action": "block", "rule": "blocklistA"},
+    {"ip": "30.1.1.3", "action": "block", "rule": "blocklistB"},
+    {"ip": "40.1.1.4", "action": "block", "rule": "blocklistA"},
+    {"ip": "50.1.1.5", "action": "block", "rule": "blocklistB"},
+]
+
+pp = pprint.PrettyPrinter(indent=4)
+```
+
+</TabItem>
+</Tabs>
+
+
+### Step 3. Add Code
+
+Add code to perform the following actions:
+
+1. Create a collection called `ddos` to which we will subscribe. If a collection by that name already exists, the existing collection is used instead.
+1. Add `data` to the collection, then subscribe to the collection. In this example, we are adding IP addresses to block.
+1. Delete `data` from the collection. If the collection becomes empty, the collection is deleted too.
```

Comment:
```suggestion
1. Delete `data` from the collection. If the collection becomes empty, then the collection is deleted too.
```

Reply:
I don't think that's what is happening. The step implies that the collection becoming empty is what triggers the deletion. In the code, though, the script is truncating the collection (emptying it of all records), then deleting it.

@koshy-thomas, am I correct here?

Also, isn't truncating before deleting an unnecessary extra step?

Reply:
Yeah you can remove the truncate command, it won't make any difference.. unless you want to show the user how he can truncate the collection via the sdks but still this is not right place to show this.. so feel free to remove the truncate command.